ui/src/layout/Events/EventDetails/index.js

```diff
@@ -3,16 +3,18 @@ import { useParams, useRouteMatch } from 'react-router-dom';
 import BackRouteButton from 'components/BackRouteButton';
 import Title from 'components/Title';
 import TabbedPageContainer from 'components/TabbedPageContainer';
+import PageContainer from 'components/PageContainer';
 import Loader from 'components/Loader';
 import { useFetch } from 'hooks';
 import { formatDate } from 'utils/utils';
 import Details from './Details';
-import SpecDiff from './SpecDiff';
+import Specs from './Specs';
+import { getPlugins, PLUGIN_TYPES } from 'plugins';
```

Comment:
move to top

Reply:
Not sure I understand.  Move to the top of where, the file?